src/main/java/org/spongepowered/mod/SpongeCoremod.java

```diff
@@ -116,10 +116,8 @@ public String getSetupClass() {
 
     @Override
     public void injectData(Map<String, Object> data) {
-        if ((Boolean)data.get("runtimeDeobfuscationEnabled")) {
```

Comment:
Yes - previously, `MixinErrorHandler` always printed a length error message to the console. Now, it skips the message when running in development.